src/auth/screens/events.screen.js

```diff
@@ -1,30 +1,20 @@
 /* eslint react/prop-types: 0 */
-
+/* eslint-disable no-shadow */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { StyleSheet, Text, FlatList, View } from 'react-native';
-import moment from 'moment';
+import moment from 'moment/min/moment-with-locales.min';
+import I18n from 'react-native-i18n';
 
 import { LoadingUserListItem, UserListItem, ViewContainer } from 'components';
 import { colors, fonts, normalize } from 'config';
 import { emojifyText, translate } from 'utils';
 import { getUserEvents } from '../auth.action';
 
-moment.updateLocale('en', {
-  relativeTime: {
-    past: '%s',
-    s: '%ds',
-    m: '%dm',
-    mm: '%dm',
-    h: '%dh',
-    hh: '%dh',
-    d: '%dd',
-    dd: '%dd',
-    M: '%dmo',
-    MM: '%dmo',
-    y: '%dy',
-    yy: '%dy',
-  },
+const language = I18n.locale.substr(0, 2);
```

Comment:
I solved this problem, you can see in [this commit](https://github.com/gitpoint/git-point/pull/193/commits/bc73e882087340041f4418ccfef1b1a74d405ae7). Now the configuration of the locale is made in `App.js`, as it probably should be there and localization works as I described it in [my comment](https://github.com/gitpoint/git-point/pull/193/commits/bc73e882087340041f4418ccfef1b1a74d405ae7).

In my opinion, it turned out pretty good, as usual, I'm waiting for your feedback :raised_hands: